src/Main.java

```diff
@@ -9,7 +9,7 @@ public static void main(String[] args) {
         try {
             firstShoppersCart.addProduct( sneakers );
```

Comment:
Are you adding more than one product here? Looks like this is the case looking at the var naming.